bin/build-plugin-zip.sh

```diff
@@ -40,6 +40,9 @@ warning () {
 }
 
 status "💃 Time to release WooCommerce Admin 🕺"
+if [ $DRY_RUN ]; then
+  warning "This is a dry run, nothing will be pushed up to Github, it will only generate the two zips"
```

Comment:
Nitpick but maybe this could just be "This is a dry run, nothing will be pushed up to Github, it will only generate zip files."